src/features/events_list/components/FeedSelector/FeedSelector.tsx

```diff
@@ -26,10 +20,11 @@ const FeedSelectorComp = () => {
     [setCurrentFeed, scheduleAutoSelect],
   );
 
-  return (featureFlags[FeatureFlag.FEED_SELECTOR] ||
-    featureFlags[FeatureFlag.EVENTS_LIST__FEED_SELECTOR]) &&
-    eventFeeds &&
-    eventFeeds.length > 1 ? (
+  if (eventFeeds?.length < 2) {
```

Comment:
To prevent compare `undefined` and 2 we can use 

`eventFeeds?.length ?? 0 < 2`